src/docs/Contributing/Designers/NamingConvention/index.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Naming convention'
+order: 5
+---
+
+> Mozaic is a product used by a lot of people. When so many people work together from different places on such a big project, you need to establish some ground rules that everyone needs to follow. These rules must be followed across every tool your team is using to create consistency and simplify the collaboration process.
+
+We chose to use english in all our files to be understandable by the most. We also chose to use the PascalCase standard in our prototyping tools.
+
+## File Naming Conventions
+
+We use two letters prefixes to name our files, this two letters depending on the pattern type and look like that: **_[XX] File name.sketch_**
+
+| Prefix   | Pattern type        | Example                                     |
+| -------- | ------------------- | ------------------------------------------- |
+| **[AA]** | Styles              | Colors, icons, fonts, shadows, ...          |
+| **[BA]** | Simple Components   | Buttons, links, tags, forms, ...            |
+| **[BB]** | Complex Components  | Notifications, accordions, popins, tabs ... |
+| **[ZZ]** | Specific librairies | Tools, documentation, ...                   |
+
+## Sketch
+
+Please follow these rules to have the same architecture and standards in every library. This will be very helpful for the design teams to understand every file in Mozaic.
+
+### Symbol categorisation
+
+We use this categorisation to create consistency in our libraries:
+
+| Categrory             | Usage                                                                                                                                                    |
+| --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **👾 For developers** | These symbols should not be used in your designs. They are meant to share the different states of a component with the development team.                 |
+| **🛠 ToolKit**         | This section contains the symbols used to build the final symbols. They are not meant to be used directly but to be accessible through symbol overrides. |
+| **Component**         | This is where you'll find the ready to use symbols of the library.                                                                                       |
```

Comment:
It's marked component, but on the screenshot, we have "buttons". 

We could use "Patterns"  (According to our glossary) 
![image](https://user-images.githubusercontent.com/5315842/66823410-7e772f00-ef46-11e9-987b-ac3df48b2e6e.png)